libs/device/models/src/lib/device-communication-event.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const ApiSerialPortEvents = {
+  DataReceived: "api-serial-port-data-received",
+} as const
+
+export type ApiSerialPortEvent =
+  (typeof ApiSerialPortEvents)[keyof typeof ApiSerialPortEvents]
+
+export const ApiSerialPortToRendererEvents = {
+  Connected: "api-serial-port-connected",
+  Disconnected: "api-serial-port-disconnected",
```

Comment:
🟡 
I suggest changing 
```suggestion
  Detached: "api-serial-port-detached",
```
because the term "Disconnected" can be misleading. In the case of an application allowing disconnection of the device, the device remains physically attached to the USB port, corresponding to the "Attached" state.

This approach more accurately reflects the state where the device is visible in the USB stack but has not yet initiated a connection, even when the user disconnects the device in the application layer.

Reply:
even more correct name would be `api-serial-port-closed` because the event is emitted as a response for serial port `close` event  :)